pex/bin/pex.py

```diff
@@ -1375,6 +1375,8 @@ def do_main(
                         ),
                         V=options.verbosity,
                     )
+                if scie_configuration.options.scie_only:
+                    os.unlink(pex_file)
```

Comment:
I assume this means that in certain cases of early failure the .pex file will be left behind as cruft. I imagine that is fine, but @martina-oefelein may have thoughts.

Reply:
Yes, exactly. The existing scie creation code also goes to lengths to preserve the temporary lift.toml it generates when there is a scie building failure; so this intent to leave behind the inputs to the failure for inspection is consistent. In the lift.toml case the fact that it is being plopped out is actually printed to the console.

Reply:
Sorry, I was offline. Yes, I think leaving the .pex file behind on failure is fine.